src/server.js

```diff
@@ -71,8 +71,14 @@ module.exports = function() {
 
 	manager.sockets = sockets;
 
-	var protocol = config.https.enable ? "https" : "http";
-	log.info("The Lounge v" + pkg.version + " is now running on", protocol + "://" + (config.host || "*") + ":" + config.port + "/", (config.public ? "in public mode" : "in private mode"));
+	let protocol = config.https.enable ? "https" : "http";
+	let host = config.host || "*";
+	if (Helper.getGitCommit()) {
```

Comment:
I'm not even sure this different message is needed, could always just be printing the `else` statement.